modules/core/src/main/scala/org/scalasteward/core/update/UpdateService.scala

```diff
@@ -127,29 +126,34 @@ final class UpdateService[F[_]](
         val dependencies = repoCache.dependencies
 
         dependencies.traverse { dependency =>
-          findUpdateState(repo, repoCache.sha1, dependency, updates1)
+          findUpdateState(repo, repoCache, dependency, updates1)
         }
       case None => List.empty[UpdateState].pure[F]
     }
 
   def findUpdateState(
       repo: Repo,
-      sha1: Sha1,
+      repoCache: RepoCache,
       dependency: Dependency,
       updates: List[Update.Single]
   ): F[UpdateState] =
     updates.find(UpdateService.isUpdateFor(_, dependency)) match {
       case None => F.pure(DependencyUpToDate(dependency))
       case Some(update) =>
-        pullRequestRepo.findPullRequest(repo, dependency, update.nextVersion).map {
-          case None =>
-            DependencyOutdated(dependency, update)
-          case Some((uri, _, state)) if state === Closed =>
-            PullRequestClosed(dependency, update, uri)
-          case Some((uri, baseSha1, _)) if baseSha1 === sha1 =>
-            PullRequestUpToDate(dependency, update, uri)
-          case Some((uri, _, _)) =>
-            PullRequestOutdated(dependency, update, uri)
+        repoCache.maybeRepoConfig.map(_.updates.keep(update)) match {
```

Comment:
Can Steward update a dependency again if the dependency is removed from ignore list in repo config?

Reply:
Yes. If the repo config does not ignore it any longer `_.updates.keep` will return a `Right(_)` there.